cli/commands/fix-paths/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Fix Paths
+permalink: /cli/commands/fix-paths/
+---
+
+* TOC
+{:toc}
+
+```
+kbc fix-paths [flags]
+```
+
+Ensure that all local paths match configured naming according to the manifest file (located in `.keboola/manifest.json).
+
+The command unifies names of configurations, rows, and code blocks in transformations in the manifest, `config.json` 
+and directory names. It would be useful also after [change of naming](/cli/configuration/#naming). 
+
+## Options
+
+`--dry-run`
+: Preview all paths that would be affected
+
+[Global Options](/cli/commands/#global-options)
+
+## Examples
+
+When you change the `config` [naming](/cli/configuration/#naming) to not include ids
```

Comment:
Tu mal mal byt skor priklad, ... ze mam nejaky config, premenujem ho v `meta.json`, spustim `fix-paths`.
Adresar sa mi premenuje podla `name`. + zmienka, ze sa to deje automaticky po `pull`.

Reply:
A ještě jsem se na to ptal. 🙈 Doplním.